src/common/models/AudioRewards.ts

```diff
@@ -79,3 +80,15 @@ export type FlowSessionEvent =
   | FlowSessionResumeEvent
   | FlowSessionPassEvent
   | FlowSessionFailEvent
+
+/**
+ * @deprecated amounts should be pulled in directly from user challenges instead
```

Comment:
Still need config for the notifications.

Moved this here so at least notifications can share one source of truth between web/mobile

Reply:
nice - might be nice to include this context in the comment so we remember why we're keeping this around